### PR TITLE
Duplicate tabs using the scroll wheel

### PR DESCRIPTION
**Pull request by ericek111**
This has been a minor annoyance of mine, along with the notebook tab scroll removed from GTK3, because why make stuff convenient for the end user... Now we can have both!

### CODE REVIEW

**Review by lukefromdc — APPROVED** [review]
Builds and runs fine, works as intended: middle click on reload button in the toolbar opens and reloads same window in a new tab, left click on reload button just reloads the same window, same as before.

I noticed by a mate-search-tool search for "caja_window_reload" that this function (as well as being rewritten) is moved to caja-window.c where one would have expected to find it anyway as it is already referenced in caja-window.h and was not previously used in caja-window-manage-views.c which raises questions about why it was there in the first place.

**Comment by raveit65** [comment]
@yetist 
Can you please take a look at archlinux build issue? Seems it's the same problem as in other repos.
I don't have archlinux installed, in result i can't compile it locally to reproduce the build issue.

**Comment by ericek111** [comment]
Just FYI, I use Arch Linux and the repo at the time of this PR compiled fine. It only errors out because of one more file present after cleanup:
```
ERROR: files left in build directory after distclean:
./docs/reference/libcaja-extension/libcaja-extension.actions
```

(I have no idea what this is or how to fix it.)

**Comment by raveit65** [comment]
See https://github.com/mate-desktop/mate-panel/pull/1151 for more infos about build issue.
Does `make distcheck` fail when you compile in your box?
Maybe the same fix helps here?
https://github.com/mate-desktop/mate-panel/pull/1151/commits/571787143d4d6dc7f9b39acd97fbd99acdaf4354

**Comment by raveit65** [comment]
@ericek111 
Please rebase against master to get https://github.com/mate-desktop/caja/commit/02375fa336ce11c0cc5949691babc03d9bfb9023 in this branch.
This fixes the build issue with travis CI.

**Comment by ericek111** [comment]
Woops, sorry. Yes, rebased it builds fine on Arch with the latest updates.

File: src/caja-window.c [hunk 1]
```diff
@@ -476,6 +483,29 @@ caja_window_sync_allow_stop (CajaWindow *window,
     }
 }
 
+void
+caja_window_reload (CajaWindow *window, gboolean new_tab)
+{
+    CajaWindowSlot *slot;
+    GList *selection;
```
Comment (raveit65) [inline src/caja-window.c:490]: Build warning. https://travis-ci.org/github/mate-desktop/caja/jobs/740579262#L18320
```
caja-window.c:490:12: warning: unused variable 'selection' [-Wunused-variable]

    GList *selection;

           ^
```

File: src/caja-window.c [hunk 2]
```diff
@@ -2220,6 +2250,6 @@ caja_window_class_init (CajaWindowClass *class)
                                   "prompt-for-location", 1,
                                   G_TYPE_STRING, "/");
 
-    class->reload = caja_window_reload;
+    class->reload = caja_window_reload_signal;
```
Comment (raveit65) [inline src/caja-window.c:2253]: Another build warning https://travis-ci.org/github/mate-desktop/caja/jobs/740579262#L18971
```
caja-window.c:2253:19: warning: incompatible pointer types assigning to 'void (*)(CajaWindow *)' (aka 'void (*)(struct CajaWindow *)') from 'gboolean (CajaWindow *)' (aka 'int (struct CajaWindow *)') [-Wincompatible-pointer-types]

    class->reload = caja_window_reload_signal;

                  ^ ~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Review by lukefromdc — APPROVED** [review]
Just retested, builds and runs fine again and works as intended. 
Caja-window-manage-views.o and caja-window-menus.o now build with no build warnings at all. We still get the usual torrent of deprecation warnings from building caja-window.c of course but I wasn't able to find anything else on searching for them in a paste of the build text into pluma.

When installed, this does not put any runtime warnings in a terminal from which sudo caja is being run after installation.

**Review by raveit65 — APPROVED** [review]
LGTM,
it works like a charm,
thanks.

**Pull request merged**